.github/workflows/sim.yaml

```
@@ -0,0 +1,92 @@
+name: Simulation Tests Workflow
```

```suggestion
name: sim
```

I think it's good for the filename to match the workflow name